### DISCUSSION TITLE
how to build to DEB RPM SNAP and APPIMAGE

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#598] [Q&A]
Is there a way to build nicegui to DEB RPM snap and appimage . Agara works on the desktop .

**Comment by rodja** [comment]
Maybe you can use [fpm](https://github.com/jordansissel/fpm) to create linux packages from a [PyInstaller build](https://nicegui.io/reference#package_for_installation)?
